xml/CRSChannelCode.xml

```diff
@@ -352,6 +352,13 @@
                 <narrative>World Vision</narrative>
             </name>
         </codelist-item>
+        <codelist-item status="active" activation-date="2020-03-06">
```

Comment:
The activation-date here (and elsewhere) is set to the day the code ran. It’s probably possible to do better than that.